tests/setup_cfg_fmt_test.py

```diff
@@ -808,3 +815,74 @@ def test_rewrite_extras(tmpdir):
         '    hypothesis\n'
         '    pytest\n'
     )
+
+
+def test_imp_classifiers_from_tox_ini(tmpdir):
```

Comment:
there should be an additional test with just `envlist = pypy3` -- I suspect right now it is adding the cpython tag when it shouldn't be